docs/examples/gluster-s3-storage-template/README.md

```diff
@@ -0,0 +1,202 @@
+# Gluster S3 docker container using template
```

Comment:
Change this to something like `Gluster S3 (gluster-object) as native app on OpenShift`.